src/deploy/osp_deployer/sah.py

```diff
@@ -319,6 +319,12 @@ def upload_director_scripts(self):
                          remote_file)
         self.run("chmod 777 /root/deploy-director-vm.sh")
 
+    def upload_powerflex_scripts(self):
+        remote_file = "/root/deploy-powerflexgw-vm.sh"
+        self.upload_file(self.settings.deploy_powerflexgw_vm_sh,
+                         remote_file)
+        self.run("chmod 777 /root/deploy-powerflexgw-vm.sh")
```

Comment:
Should 777 be just +x? And sudo?

Reply:
No need for sudo as we use root. We're doing it the same way we create the director.